src/plugins/workspace/public/plugin.ts

```diff
@@ -148,9 +148,14 @@ export class WorkspacesPlugin implements Plugin<{}, {}, WorkspacesPluginSetupDep
     workspace: WorkspaceAttribute | null | undefined,
     allNavLinks: ChromeNavLink[]
   ) {
-    if (!workspace) return allNavLinks;
+    const outsideWorkspaceCategoryIds = new Set<string>(['management', 'workspace']);
```

Comment:
Could we reference the category id from `src/core/utils/default_app_categories.ts`?